src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,25 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){} // 2 violations , '{' is not followed and preceded by whitespace.
+ *     public static void main(String[] args) {
+ *         if (foo) { // ok
+ *             // body
+ *         }
+ *         else{ // violation
+ *             // body
+ *         }
+ *         for (int i = 1; i &gt; 1; i++) {} // 2 violations ,
+ *                                             // '{' is not followed and preceded by whitespace.
```

Comment:
`{` is preceded by space , so this comment is not correct, please look at comment I proposed to you originally.

Reply:
@romani Done